user_guides/jaxrs/src/main/jbake/content/rebuild.inc

```diff
@@ -98,12 +98,12 @@ contains the following elements:
 <web-app version="2.5" xmlns="http://java.sun.com/xml/ns/javaee" \
```

Comment:
version to 5.0, can replace other occurrences of http://java.sun.com/xml/ns/javaee to https://jakarta.ee/xml/ns/jakartaee too.